app/models/job_queue.rb

```diff
@@ -61,7 +62,11 @@ def debug
       grouped[i[:queue]] << i[:job_execution]
     end
 
-    [@executing, grouped]
+    if staggering_enabled?
+      [@executing, grouped, staggered_jobs_debug]
+    else
+      [@executing, grouped]
```

Comment:
prefer more dry `list = [a,b]; list << c if c?; list`

Reply:
maybe just always return staggered since they will be empty anyways and then not display them when empty